src/Features/CSharp/Portable/MakeLocalFunctionStatic/PassInCapturedVariablesAsArgumentsCodeFixProvider.cs

```diff
@@ -101,8 +101,7 @@ protected override Task FixAllAsync(Document document, ImmutableArray<Diagnostic
 
             foreach (var localFunction in localFunctions)
             {
-
-                if (MakeLocalFunctionStaticHelper.TryGetCaputuredSymbolsAndCheckApplicability(localFunction, semanticModel, out var captures))
+                if (MakeLocalFunctionStaticHelper.CanMakeLocalFunctionStaticByRefactoringCaptures(localFunction, semanticModel, out var captures))
```

Comment:
This fixer is only available on static local functions, so all of the existing tests already validate that nothing is broken.